tests/python_client/bulk_load/bulk_load_data.py

```diff
@@ -7,8 +7,8 @@
 from minio_comm import copy_files_to_minio
 
 # TODO: remove hardcode with input configurations
-minio = "minio_address:port"     # minio service and port
-bucket_name = "milvus-bulk-load"  # bucket name of milvus is using
+minio = "localhost:9000"     # minio service and port
+bucket_name = "a-bucket"  # bucket name of milvus is using
```

Comment:
`localhost` might works in local test. Maybe will cause the workflow to fail in CI env.

Reply:
Done.